ls1intum/Artemis
# Improve impact warning view after user feedback

### Checklist

### Motivation and Context


The impact warning was noticed from tutors grading the first text submissions with positive remarks. However, they initially perceived the warning as a mistake of theirs rather then a helpful information. Based on suggestions a new more friendly view is offered in this PR through minor CSS changes.

### Description

Changed the impact warning icon from **warning** to **info** and added tooltip on the text with more information. Text size could also be decreased. 

### Steps for Testing

1. Log in to Artemis
2. Navigate to a TextExercise submission with similiar submissions
3. Click and add points to a block to see the new warning appear.

### Altrenative Solutions:



![zzzz1](https://user-images.githubusercontent.com/18335489/122491762-de0fc280-cfe4-11eb-9347-45ddbdd0b634.PNG)



### Current Implementation:


![c2](https://user-images.githubusercontent.com/18335489/122302801-e68fcc80-cf02-11eb-9ed5-d23e06d3ddca.PNG)
